src/test/java/gov/healthit/chpl/aqa/pageObjects/CollectionsPages.java

```diff
@@ -41,4 +41,12 @@ public static WebElement cmsFaqLink(final WebDriver driver) {
         return element;
     }
 
+    /**
+     * Returns element holding SED Details File.
+     * @param driver WebDriver
+     * @return element holding SED Details File
+     */
+    public static WebElement sedDetailsFileButton(final WebDriver driver) {
+        return driver.findElement(By.xpath("//a[@type='button']"));
```

Comment:
This seems like a very broad xpath...

Reply:
I'd like to request id for SED Download button on SED Info collections page.